backend/migrations/20240830222936_add_nullable_season.sql

```diff
@@ -0,0 +1,13 @@
+-- Create "seasons" table
+CREATE TABLE "seasons" (
+  "id" bigserial NOT NULL,
+  "created_at" timestamptz NULL,
+  "updated_at" timestamptz NULL,
+  "deleted_at" timestamptz NULL,
+  PRIMARY KEY ("id")
+);
+-- Create index "idx_seasons_deleted_at" to table: "seasons"
+CREATE INDEX "idx_seasons_deleted_at" ON "seasons" ("deleted_at");
+-- Modify "matches" table
+ALTER TABLE "matches" ADD COLUMN "season_id" bigint NULL, ADD
```

Comment:
Skal vi gøre sådan at season ikke må være null? Du kan sætte den som not-null efter du har sat season id på i migration.

Reply:
Jeg undersøger lige. Var oprindeligt min tanke, men Gorm og Atlas ville ikke lave ændringer, da jeg fjernede nullability i Gorm

Reply:
Det er det samme med teams. Jeg lader det være lige pt.